src/views/services/list/ServiceList.tsx

```diff
@@ -37,9 +38,17 @@ const ServiceList: FC<ServiceListProps> = ({ namespace }) => {
   });
   const [data, filteredData, onFilterChange] = useListPageFilter(service);
   const columns = useServiceColumn();
+  const title = t('Services');
 
   return (
-    <>
+    <ListEmptyState<IoK8sApiCoreV1Service>
+      data={data}
+      href="#"
```

Comment:
??

Reply:
couldnt find a link, will update once we hear from docs